lua/nvim-tree/diagnostics.lua

```diff
@@ -166,17 +145,6 @@ function M.setup(opts)
   for lhs, rhs in pairs(links) do
     vim.cmd("hi def link " .. lhs .. " " .. rhs)
   end
-
-  if M.enable then
```

Comment:
Let's see how it goes without the autocommands...

Reply:
the autocmds were just moved in the nvim-tree setup for autocmds